### PR TITLE
add your-cat-section

### PR DESCRIPTION
**Pull request by lenux17**

### CODE REVIEW

File: src/DSection.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import "./DSection.css";
+
+export const DSection = () => {
+  return (
+    <div className="DSection">
+      <h1>Products #####</h1>
```
Comment (hypeofpipe) [inline src/DSection.js]: Why do you need that `####` ?

File: src/DSection.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      <p>
+        The nutritional needs of cats change throughout life, which is why #####
```
Comment (hypeofpipe) [inline src/DSection.js]: Why do you need that `####`?

File: src/DSection.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        offers delicious food, the recipe of which has been developed so that
+        your cat receives exactly what it needs at every stage of its life.
+        Thanks to the content of a specially selected composition of vitamins
+        and minerals, ##### foods help preserve your cat's health for many
```
Comment (hypeofpipe) [inline src/DSection.js]: Why do you need that `####`?

File: src/YCSection.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import "./YCSection.css";
+
+export const YCSection = () => {
```
Comment (hypeofpipe) [inline src/YCSection.js]: Rename it to `YourCatSection`

File: src/YCSection.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div className="YCSection">
```
Comment (hypeofpipe) [inline src/YCSection.js]: Rename it to `YourCatSection`

File: src/YCSection.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.YCSection p {
```
Comment (hypeofpipe) [inline src/YCSection.css]: Rename it to `YourCatSection`

File: src/YCSection.css [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    width: 50rem;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+    font-weight: 500;
+    margin-bottom: auto;
+}
+
+.YCSection h1 {
```
Comment (hypeofpipe) [inline src/YCSection.css]: Rename it to `YourCatSection`

File: src/YCSection.css [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    margin-top: auto;
+}
+
+.YCSection {
```
Comment (hypeofpipe) [inline src/YCSection.css]: Rename it to `YourCatSection`

File: src/App.js [hunk 9]
```diff
@@ -3,13 +3,17 @@ import "./App.css";
...
+import { YCSection } from "./YCSection";
+import { DSection } from "./DSection";
 
 function App() {
   return (
     <div className="App">
       <Header />
       <Links />
       <Slider />
+      <YCSection />
```
Comment (hypeofpipe) [inline src/App.js]: Rename it to `YourCatSection`

File: src/App.js [hunk 10]
```diff
@@ -3,13 +3,17 @@ import "./App.css";
 import { Header } from "./Header";
 import { Links } from "./Links";
 import { Slider } from "./Slider";
+import { YCSection } from "./YCSection";
```
Comment (hypeofpipe) [inline src/App.js]: Rename it to `YourCatSection`

File: src/DSection.js (hunk 1 above)
Comment (lenux17) [reply src/DSection.js]: there are name product(це є назва продукту, потім поміняється)

**Pull request merged**